package.json

```diff
@@ -169,6 +181,6 @@
 	},
 	"publishConfig": {
 		"access": "public",
-		"tag": "latest"
+		"tag": "next"
```

Comment:
Should this be changed back to "latest" ?
Note that you probably shouldn't publish "next" releases without a prerelease tag, otherwise npm automatically uses the most recent release because of the `^` in the version string

Reply:
> Note that you probably shouldn't publish "next" releases without a prerelease tag, otherwise npm automatically uses the most recent release because of the ^ in the version string

Should it? Coz, I have used the [npm calculator](https://semver.npmjs.com/) to see if the `next` tag is being picked or not. Am I missing something here?

<img width="848" alt="CleanShot 2021-08-25 at 18 06 24@2x" src="https://user-images.githubusercontent.com/1706381/130791566-ec2c8a78-760b-4296-aed7-814c951ae746.png">

Reply:
sorry you're right. It's the first time I see this behavior, I was certain that tags had 0 influence on semver ranges.

Reply:
Yeah, maybe it is something npm specific. Coz semver does recommend appending preview or alpha to the version number. 

will do it from next time

Reply:
I still think it would be better to not use normal version numbers for prereleases, but it's less important if it doesn't have any impact on users

Reply:
> Should this be changed back to "latest" ?

Regarding this, I think we should change it post merge inside the `develop` branch?